README.md

```diff
@@ -1 +1,42 @@
-# Hello_Microverse_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
```

Comment:
- [ ] Here you could also add a short description of what the project is all about. This too will add professionalism to your Readme file.